velox/docs/functions/spark/datetime.rst

```diff
@@ -83,6 +83,12 @@ These functions support TIMESTAMP and DATE input types.
         SELECT get_timestamp('1970-01-01', null);  -- NULL
         SELECT get_timestamp('2020-06-10', 'A');  -- (throws exception)
 
+.. spark:function:: hour(string/timestamp) -> integer
+
+    Returns the hour of ``string/timestamp``.::
```

Comment:
As this is to document the spark function implemented in velox, I think we should only keep timestamp type to align with the function signature.

Reply:
Thanks, changed.